aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageAdapter.java

```diff
@@ -521,6 +521,70 @@ public void query(
         });
     }
 
+    public <T extends Model> void delete(
```

Comment:
```suggestion
    @Override
    public <T extends Model> void delete(
```

Any way we can achieve this functionality on the storage adapter without forking/cloning a new method instance?